### PR TITLE
add note to concepts section

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/camunda-docs/issues/3327.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/concepts/process-instance-creation.md [hunk 1]
```diff
@@ -114,6 +114,10 @@ By default, the instruction starts before the given element. This means input ma
 Multiple instructions can be provided to start the process instance at more than one element.
 You can activate the same element multiple times inside the created process instance by referring to the same element id in more than one instruction.
 
+:::note
+Start instructions have the same [limitations as the process instance modification](/components/concepts/process-instance-modification.md#limitations), e.g., it is not possible to start an element in a multi-instance subprocess.
```
Comment (akeller) [inline docs/components/concepts/process-instance-creation.md:118]: This is very nit picky, but do we say "the process instance modification" or just "process instance modification?" No "the?"

Comment (korthout) [reply docs/components/concepts/process-instance-creation.md:118]: English is not my first language 😄 but if it's any help: I refer to it as `process instance modification` (without `the`).

Comment (korthout) [inline docs/components/concepts/process-instance-creation.md:118]: > e.g., it is not possible to start an element in a multi-instance subprocess.

This example may not be future-proof. There may be a time when we'll support this. Let's use `it is not possible to start at a sequence flow` instead, as we're unlikely to support that.

**Comment by christinaausley** [comment]
@korthout @akeller I've addressed your comments across all files that reference this note. Ready for final review 👍

File: docs/apis-tools/zeebe-api/gateway-service.md [hunk 2]
```diff
@@ -226,7 +226,7 @@ Only processes with none start events can be started through this command.
 :::
 
 :::note
-Start instructions have the same [limitations as the process instance modification](/components/concepts/process-instance-modification.md#limitations), e.g., it is not possible to start an element in a multi-instance subprocess.
+Start instructions have the same [limitations as process instance modification](/components/concepts/process-instance-modification.md#limitations), e.g., it is not possible to start an element at a sequence flow.
```
Comment (korthout) [inline docs/apis-tools/zeebe-api/gateway-service.md:229]: >it is not possible to start an element at a sequence flow.

@christinaausley I think this is not correct. An element is not started at a sequence flow. Rather, a sequence flow is an element. 

IMO, `it is not possible to start at a sequence flow` is better. We actually mean to say `it is not possible to start a process instance at a sequence flow`

**Review by korthout — APPROVED** [review]
Thanks @christinaausley LGTM 💯

**Pull request merged**